README.md

```diff
@@ -125,31 +125,12 @@ The distance bias (in 3D units) to use for LOD calculations. Positive values
 will improve performance at the cost of visual quality, whereas negative values
 will improve visual quality at the cost of performance.
 
-### `lod/refresh_rate`
+### `lod/refresh_threshold_ms`
 
-*Default:* `0.25`
+*Default:* `10`
```

Comment:
10 milliseconds seems quite high as a default value to me. It essentially means you can't reach 120+ FPS on high-end machines. I think 5 milliseconds would be a better default, or maybe even 4 milliseconds. Remember that other game logic has to run on the CPU as well :slightly_smiling_face:

Reply:
True. I'll lower it.